### PR TITLE
Feat: Added Results Page and questions to DB

### PR DESCRIPTION
**Pull request by sainath7878**
Added Results Page

Files to review

- [src/components/resultCard/ResultCard.jsx](https://github.com/sainath7878/joddquiz/pull/4/files#diff-7e81838301b386be1511d90774bb3a37c68e42764dd619f416e176b498eb6224)
- [src/pages/result/ResultPage.jsx](https://github.com/sainath7878/joddquiz/pull/4/files#diff-e19a9d529840dd5c100c9aaa843b51d6f956a850d821960bf618bda6b0707320)
- [src/reducer/quizReducer.js](https://github.com/sainath7878/joddquiz/pull/4/files#diff-cb3fe1c5da553865be877bdbeaf374760d3abdadb82e86ac952f0adfeeff1bc1)

### CODE REVIEW

File: src/pages/result/ResultPage.jsx [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+      <h1 className="fs-l secondary-text">Result</h1>
+
+      {quiz && (
+        <h2 className="fs-ml">
+          Final Score: {score}/{quiz.totalScore}
+        </h2>
+      )}
+      {quiz &&
+        quiz?.mcqs.length > 0 &&
+        quiz.mcqs.map((question, index) => {
```
Comment (anik31) [inline src/pages/result/ResultPage.jsx:24]: appreciation - good that you're checking for falsy values

File: src/components/resultCard/ResultCard.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+function ResultCard({ questionObj, questionNumber }) {
+  const { question, options, answer } = questionObj;
+
+  const {
+    quizState: { questionAnswers },
+  } = useQuiz();
+
+  console.log(options, questionNumber);
+
```
Comment (anik31) [inline src/components/resultCard/ResultCard.jsx]: suggestion - remove log if not required

**Review by anik31 — APPROVED** [review]
Good job, the code is clean and functionality is also working fine. Liked that you've used loaders and toast.

File: src/components/resultCard/ResultCard.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <h1 className="fs-m">{question}</h1>
+        <ul className={styles.list}>
+          {options.map((option) => {
+            console.log(
+              option,
+              answer,
+              questionNumber,
+              questionAnswers,
+              questionAnswers[questionNumber]
+            );
```
Comment (BalajiHarinath) [inline src/components/resultCard/ResultCard.jsx]: Can remove this console log if not required

Comment (sainath7878) [reply src/components/resultCard/ResultCard.jsx]: Yeah, will remove!

**Review by BalajiHarinath — APPROVED** [review]
Apart from the suggested change everything looks good.

**Pull request merged**